### PR TITLE
Qingwei/global user input lock

### PR DESCRIPTION
**Pull request by qingweibinary**

### CODE REVIEW

**Comment by qingweibinary** [comment]
@borisyankov OK to merge

I prevent user input to accumulate by applying a half transparent div right after user make purchase, so any user tap after that will be ignore, until purchase call finished

File: src/loading-view/TransparentDiv.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React, { PureComponent } from 'react';
+
+export default class TransparentDiv extends PureComponent {
+    render() {
+        return (
+            <div
+                style={{
+                    height: '100vh',
```
Comment (borisyankov) [inline src/loading-view/TransparentDiv.js]: That is literally a css and a < div className="invisible" > or something ;)

Comment (qingweibinary) [inline src/loading-view/TransparentDiv.js]: Amended =)

File: Gulpfile.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ const sass = require('gulp-sass');
 // const zip = require('gulp-vinyl-zip');
 
 const files = {
-    dist: './dist',
+    dist: './www',
```
Comment (borisyankov) [inline Gulpfile.js]: Hmmm. Not sure if correct?

Comment (qingweibinary) [inline Gulpfile.js]: You're right, I'll close this

**Pull request closed without merging**